submissions/kt-std/js-dom/script.js

```diff
@@ -0,0 +1,145 @@
+        period: '687 Earth days',
+        position: '04',
+    },
+];
+
+document
+    .querySelector('#menu-btn')
+    .addEventListener('click', function display(e) {
+        menu.classList.toggle('visible');
+    });
+
+document
+    .querySelector('#menu')
+    .addEventListener('click', function display(e) {
+    const sectionName = e.target.innerHTML.toLowerCase();
+    index = CONTENT.findIndex((el) => el.heading.toLowerCase() === sectionName);
+    setData(index);
+});
+
+const addAnimation = (element, ...animationOptions) =>
+    element.animate(...animationOptions);
+
+const updateBackground = (element, path, id) => {
+    element.style.backgroundImage =
+        window.innerWidth < SCREEN_WIDTH
+            ? `url('${path + id + '_small'}.jpg')`
+            : `url('${path + id}.jpg')`;
+    addAnimation(
+        element,
+        [
+            {
+                opacity: 0.1,
+            },
+            {
+                opacity: 1,
+            },
+        ],
+        {
+            duration: 500,
+        }
+    );
+};
+
+const setInnerData = (element, data) => (element.innerHTML = data);
+
+const setData = (index) => {
+    const image = document.getElementById('tabImage'),
+        heading = document.querySelector('.tab__header'),
+        headerMenu = document.querySelector('.header'),
+        container = document.querySelector('.container'),
+        paragraph = document.querySelector('.tab__paragraph'),
+        period = document.querySelector('.statistics__text'),
+        position = document.querySelector('#position');
+
+    updateBackground(image, ASSETS_PATH, CONTENT[index].heading.toLowerCase());
+    setInnerData(heading, CONTENT[index].heading);
+    setInnerData(position, CONTENT[index].position);
+    setInnerData(paragraph, CONTENT[index].paragraph);
+    addAnimation(
+        container,
+        [
+            {
+                opacity: 0,
+            },
+            {
+                opacity: 1,
+            },
+        ],
+        {
+            duration: 600,
+        }
+    );
+
+    setInnerData(period, CONTENT[index].period);
+    if (window.innerWidth > 800) {
+        updateBackground(
+            headerMenu,
+            ASSETS_PATH,
+            `aside_${CONTENT[index].heading.toLowerCase()}`
+        );
+    }
+};
+
+window.addEventListener('resize', function(e) {
+    const image = document.querySelector('.header');
+    if (window.innerWidth <= SCREEN_WIDTH) {
+        image.style.backgroundImage = 'none';
+    } else {
+        updateBackground(
+            image,
+            ASSETS_PATH,
+            `aside_${document
+                .querySelector('.tab__header')
+                .textContent.toLowerCase()}`
+        );
+    }
+});
+
+const renderContent = (data) => {
+    const menu = document.getElementById('menu'),
+        listItemClasses = ['menu__item', 'flex', 'center', 'full'];
+    fragment = new DocumentFragment();
```

Comment:
use const for this variable

Reply:
Constant is now used, therefore this variable is not global anymore